src/net.cpp

```diff
@@ -2189,7 +2189,30 @@ void CConnman::ThreadDNSAddressSeed()
     std::vector<std::string> seeds = m_params.DNSSeeds();
     Shuffle(seeds.begin(), seeds.end(), rng);
     int seeds_right_now = 0; // Number of seeds left before testing if we have enough connections
-    int found = 0;
+    int target_outbound_connections = 2;
+    int outbound_connection_count = 0;
+
+    auto start = NodeClock::now();
+    if (gArgs.IsArgSet("-seednode")) {
+        LogPrintf("-seednode enabled. Trying the provided seeds before defaulting to the dnsseeds.\n");
```

Comment:
Only additional nit is that DNS seeds and fixed seeds output the number of seconds, might be nice & consistent to do so here too. Consider breaking out a ```constexpr std::chrono::seconds SEEDNODE_TIMEOUT = 30s;``` and `%d`-ing it into the message.

Compare with DNS seeds:
```
Waiting 300 seconds before querying DNS seeds.
```
And fixed seeds:
```
Adding fixed seeds as 60 seconds have passed and addrman is empty for at least one reachable network
```